go.mod

```diff
@@ -3,10 +3,11 @@ module github.com/vshn/stardog-userrole-operator
 go 1.13
 
 require (
-	github.com/go-logr/logr v0.1.0
+	github.com/go-logr/logr v0.2.0
 	github.com/onsi/ginkgo v1.12.1
 	github.com/onsi/gomega v1.10.1
-	k8s.io/apimachinery v0.18.6
+	k8s.io/api v0.19.3
+	k8s.io/apimachinery v0.19.3
```

Comment:
Please stick to `v0.18.6` for now, the Operator SDK 1.1 is not ready for 0.19 yet

Reply:
Updated that already